packages/account/src/Components/form-body-section/form-body-section.tsx

```diff
@@ -1,23 +1,62 @@
 import React from 'react';
 import { Text } from '@deriv/components';
 import { isMobile } from '@deriv/shared';
+import classNames from 'classnames';
+import { Localize } from '@deriv/translations';
 
 export type TFormBodySection = {
+    /**
+     *  A boolean value indicating whether the section has a side note.
+     */
     has_side_note?: boolean;
+    /**
+     * The side note to display next to the main content.
+     */
     side_note?: string | React.ReactElement;
+    /**
+     * The position of the side note relative to the main content.
+     * @default 'left'
+     * */
+    side_note_position?: 'left' | 'right';
 };
 
-const FormBodySection = ({ children, has_side_note, side_note }: React.PropsWithChildren<TFormBodySection>) => {
+/**
+ *  A component that renders a form body section with an optional side note.
+ *
+ *  This component renders a section of a form body. It can optionally display a side note
+ *  and allows reversing the order of the side note and the main content.
+ *  @returns {JSX.Element} A React component that renders a form body section.
+ */
+const FormBodySection = ({
+    children,
+    has_side_note,
+    side_note,
+    side_note_position = 'left',
+}: React.PropsWithChildren<TFormBodySection>): JSX.Element => {
     if (has_side_note) {
         return (
-            <div data-testid='dt_side_note_container' className='account-form__section'>
-                <div className='account-form__section-side-note'>
+            <div
+                data-testid='dt_side_note_container'
+                className={classNames('account-form__section', {
+                    'account-form__section--reversed': side_note_position === 'right',
+                })}
+            >
+                <div
+                    className={classNames('account-form__section-side-note', {
+                        'account-form__section-side-note__example': typeof side_note !== 'string',
+                    })}
+                >
                     {typeof side_note === 'string' ? (
                         <Text color='less-prominent' size={isMobile() ? 'xxs' : 'xs'} data-testid='side-note-text'>
                             {side_note}
                         </Text>
                     ) : (
-                        side_note
+                        <React.Fragment>
+                            <Text as='p' size={isMobile() ? 'xxs' : 'xs'} weight='bold'>
```

Comment:
**NIT:**: Can we use is_mobile from store?

Reply:
Is there a specific reason to use `is_mobile` from store? Why can't we use this `isMobile` function?

Reply:
I tried using `is_mobile` from store and ran into an issue that `useStore must be used in StoreProvider` coz this component is being compiled and exported to core to be used during account signup